command/src/main/java/com/jagrosh/jdautilities/command/AnnotatedModuleCompiler.java

```diff
@@ -20,14 +20,14 @@
 
 /**
  * A "compiler" for {@link java.lang.Object Object}s that uses {@link java.lang.annotation.Annotation Annotation}s
- * as helpers for creating {@link com.jagrosh.jdautilities.command.Command Command}s.
+ * as helpers for creating {@link Command Command}s.
```

Comment:
This breaks JDA Documentation standards, the original way this was done should have been maintained

Reply:
I honestly didn't see these. Not sure how that happened...

Reply:
When you refactor, intellij has a setting that simplifies code due to refactor.
I typically turn it off when making massive reallocations like this one